### PR TITLE
add ADDI icon

### PR DESCRIPTION
**Pull request by jforeroaddi**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Review by mariomurillo — APPROVED** [review]

**Comment by jforeroaddi** [comment]
@tauthomas01 @larouxn i don't understand why CI fail
could you tell me why

File: db/payment_icons.yml [hunk 1]
```diff
@@ -1065,4 +1065,7 @@
 -
   name: genoapay
   label: Genoapay
+-
+  name: addi
+  label: ADDI
```
Comment (larouxn) [inline db/payment_icons.yml:1070]: @jforeroaddi, it looks like the issue is this change removes `group` from the `genoapay` entry. Can we fix this?

Comment (jforeroaddi) [reply db/payment_icons.yml:1071]: @larouxn Done

**Review by larouxn — CHANGES_REQUESTED** [review]
Looks like the icon is lacking the proper rounder corners/shape? Perhaps we can mix the icon with the template we have [here](https://github.com/activemerchant/payment_icons/blob/dc8c3b638b473bba6de7c9e05c3483afcc893f2b/CONTRIBUTING.md#template) to fix it?
Screen Shot 2022-01-25 at 12 12 43

**Comment by larouxn** [comment]
Looks like the new icon now has the proper shape but is too big. Perhaps the border is external? I believe it's supposed me internal. @tauthomas01 might have some more insight. 🙇‍♂️
Screen Shot 2022-01-25 at 15 00 15

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
I can't see the border of the icon: [video demonstration](https://screenshot.click/25-08-px6zo-yaoby.mp4)

**Comment by jforeroaddi** [comment]
@larouxn  i think i solve it

**Comment by larouxn** [comment]
👋 @jforeroaddi, I just took another look. Shape still seems to be off. I suggest we try to drop the icon on top of the [template in the contribution guide](https://github.com/activemerchant/payment_icons/blob/dc8c3b638b473bba6de7c9e05c3483afcc893f2b/CONTRIBUTING.md#template) and compare local SVG against the [visa.svg](https://github.com/activemerchant/payment_icons/blob/dc8c3b638b473bba6de7c9e05c3483afcc893f2b/app/assets/images/payment_icons/visa.svg) icon.
Screen Shot 2022-01-26 at 17 44 18

**Comment by jforeroaddi** [comment]
@larouxn i think i solve it

**Review by larouxn — APPROVED** [review]
Nice work! LGTM 🎩 
Screen Shot 2022-01-27 at 19 40 39

**Comment by jforeroaddi** [comment]
@tauthomas01 could you please add your review ?

**Comment by tauthomas01** [comment]
I'm unable to view the SVG in github. The preview nor raw file doesn't work.

**Comment by jforeroaddi** [comment]
@tauthomas01 it is so strange, because in file system of my computer the image open  
image

**Comment by larouxn** [comment]
Oh true, I didn't notice. It doesn't load in GitHub. That's a bit concerning. I wonder why. 🤔 
Screen Shot 2022-01-29 at 09 47 44
Screen Shot 2022-01-29 at 09 48 48

**Comment by larouxn** [comment]
Maybe we can use the following SVG?
```SVG
<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="pi-addi"><title id="pi-addi">ADDI</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" fill="#000"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32z" fill="#fff"/><path d="M28.724 16.28h1.834v-5.313h-1.834v5.313zM30.82 9.161A1.17 1.17 0 0029.641 8c-.65 0-1.178.52-1.178 1.161a1.17 1.17 0 001.178 1.162c.65 0 1.177-.52 1.177-1.162zM24.807 14.626a1.265 1.265 0 01-1.274-1.256c0-.694.57-1.257 1.274-1.257.704 0 1.274.563 1.274 1.257 0 .693-.57 1.256-1.274 1.256zm3.113-1.256V8.303h-1.84v2.363c-.347-.22-.819-.343-1.273-.343-1.719 0-3.113 1.364-3.113 3.046 0 1.683 1.394 3.047 3.113 3.047 1.662 0 3.113-1.261 3.113-3.046zM17.979 14.626a1.265 1.265 0 01-1.274-1.256c0-.694.57-1.257 1.274-1.257.704 0 1.274.563 1.274 1.257 0 .693-.57 1.256-1.274 1.256zm3.113-1.256V8.303h-1.84v2.363c-.348-.22-.819-.343-1.273-.343-1.72 0-3.113 1.364-3.113 3.046 0 1.683 1.394 3.047 3.113 3.047 1.662 0 3.113-1.261 3.113-3.046zM11.113 11.874c-.583 0-1.08.36-1.274.867v-1.687c0-.694.57-1.256 1.274-1.256.703 0 1.274.562 1.274 1.256v1.688a1.362 1.362 0 00-1.274-.868zm1.274 1.823v2.555h1.834v-5.076c.002-.04.005-.08.005-.122 0-1.682-1.394-3.046-3.113-3.046C9.393 8.008 8 9.372 8 11.054v5.198h1.834V13.68c.192.513.692.879 1.279.879.574 0 1.065-.35 1.265-.846l.009-.017z" fill="#4E7EFF"/></svg>
```

I used the Visa SVG as a template plus the Addi logo from the icon in this PR? I have confirmed [CI passes](https://github.com/activemerchant/payment_icons/runs/4991733505?check_suite_focus=true) and it renders properly in GitHub. Feel free to copy the SVG markup above and commit it here in this PR if that works for you. 🙇‍♂️ 
Screen Shot 2022-01-29 at 09 57 06
Screen Shot 2022-01-29 at 09 58 46

**Comment by jforeroaddi** [comment]
@larouxn thanks, i hope that change work

**Review by larouxn — APPROVED** [review]
🎩 Looks good to me 🎩 
Screen Shot 2022-01-30 at 19 56 40

@tauthomas01, feel free to take a look when you have a chance.

**Review by tauthomas01 — APPROVED** [review]
Thank you @larouxn for the help. I can see it now

**Comment by jforeroaddi** [comment]
@larouxn How we can merge this PR ?

**Comment by larouxn** [comment]
👋 Hello hello, your icon(s) should be live everywhere now. Please let us know if otherwise. 🙇‍♂️

**Pull request merged**